frontend/src/e2e-test/specs/5_employee/child-information-assistance.spec.ts

```diff
@@ -648,7 +648,7 @@ describe('Child assistance need voucher coefficients for employees', () => {
     await waitUntilEqual(
       async () => await assistance.assistanceNeedVoucherCoefficients(0),
       {
-        coefficient: 'Palvelusetelikerroin 9,8',
+        coefficient: '9,8',
         validityPeriod: '10.02.2021 – 21.11.2021',
         status: 'ENDED',
         actionCount: 2
```

Comment:
Add assertion that checks for the last modified timestamp and last modifier are visible in here (`assistance need voucher coefficient can be edited`)